src/varpubs/test_extractor.py

```diff
@@ -0,0 +1,15 @@
+from varpubs.hgvs_extractor import extract_hgvsp_from_vcf
+
+def main():
+    vcf_path = "/Users/ilmaytas/Desktop/annotated.vcf"
```

Comment:
Files for testing should go into a `resources` directory within the `tests` directory and can also be committed.